.github/workflows/playwright.yml

```diff
@@ -42,27 +39,23 @@ jobs:
 
       - name: Build projects and run ConfigBuilder
         run: |
-          dotnet build Plugins/BTCPayServer.RockstarDev.Plugins.CashCheckout
+          dotnet build Plugins/BTCPayServer.RockstarDev.Plugins.CashCheckout   
+          dotnet build Plugins/BTCPayServer.RockstarDev.Plugins.Payroll
           dotnet build BTCPayServer.Plugins.Tests
           dotnet build ConfigBuilder
           dotnet run --project ConfigBuilder
+      
+      - name: Install Playwright browsers
+        run: |
+          dotnet tool install --global Microsoft.Playwright.CLI
+          export PATH="$PATH:$HOME/.dotnet/tools"
+          playwright install --with-deps
```

Comment:
To me this looks fantastic now, but not sure if @NicolasDorier will insist that we do it with `docker build` the way he did in core: https://github.com/btcpayserver/btcpayserver/blob/18fca7162065823bf53a5aba71794d8dabdafc3d/BTCPayServer.Tests/docker-compose.altcoins.yml#L8

Let's see what he says after reviewing... he can also try to tweak it the way he prefers when he has time.